client_app/src/styles/shelter/emergencyAlertModal.css

```diff
@@ -0,0 +1,21 @@
+.emergencyMessage {
+  background-color: white;
+  height: 30vh;
+  width: 70vw;
+  margin-left: auto;
+  margin-right: auto;
+  display: block;
+}
+
```

Comment:
Add a larger gray margin around the emergency alert, it's slightly too small, or add a border around it like the other modals